scheduler/test/cook/test/simulator.clj

```diff
@@ -0,0 +1,586 @@
+(ns cook.test.simulator
```

Comment:
I think it is worth keeping it where it is. It is still effectively a test of cook. I will add an actual unit test to highlight that.